package.json

```diff
@@ -188,21 +190,24 @@
     "@types/mocha": "^5.2.0",
     "@types/node": "^10.7.1",
     "@types/sinon": "^5.0.2",
+    "chai": "^4.1.2",
     "decache": "^4.4.0",
+    "glob": "^7.1.3",
+    "istanbul": "^0.4.5",
     "mocha": "^5.2.0",
-    "tslint": "^5.11.0",
-    "chai": "^4.1.2",
+    "mocha-jenkins-reporter": "^0.4.0",
+    "remap-istanbul": "^0.12.0",
     "sinon": "^6.3.4",
     "sinon-chai": "^3.2.0",
+    "ts-loader": "^6.1.0",
+    "tslint": "^5.11.0",
     "typescript": "^3.0.3",
     "vscode": "^1.1.33",
-    "glob": "^7.1.3",
-    "istanbul": "^0.4.5",
-    "mocha-jenkins-reporter": "^0.4.0",
-    "remap-istanbul": "^0.12.0"
+    "webpack": "^4.40.1",
+    "webpack-cli": "^3.3.8"
   },
   "dependencies": {
-    "fabric8-analytics-lsp-server": "0.1.56",
+    "fabric8-analytics-lsp-server": "file:../fabric8-analytics-lsp-server/output/fabric8-analytics-lsp-server-0.0.0-development.tgz",
```

Comment:
This is a temp fix intended for dev purpose, will be removed once we get latest lsp module.